<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureGroupConfigForm</class>
 <widget class="QWidget" name="TemperatureGroupConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="temperatureCouplingGroupLabel">
       <property name="text">
        <string>Coupling Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="temperatureCouplingGroup"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="temperatureLabel">
       <property name="text">
        <string>Temperature (K)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="temperature">
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="temperatureUpdateIntervalLabel">
       <property name="text">
        <string>Update Interval  (ps)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="temperatureUpdateInterval">
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" alignment="Qt::AlignTop">
    <widget class="QToolButton" name="removeTemperatureGroup">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="remove"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
